test/run_test.py

```diff
@@ -592,10 +592,16 @@ def print_to_file(s):
         signal_name = f" ({SIGNALS_TO_NAMES_DICT[-ret_code]})" if ret_code < 0 else ""
         print_to_file(f"Got exit code {ret_code}{signal_name}")
 
-        # Read what just failed
-        with open(
+        stepcurrent_file = (
             REPO_ROOT / ".pytest_cache/v/cache/stepcurrent" / stepcurrent_key
-        ) as f:
+        )
+        if not stepcurrent_file.exists():
+            # Pytest didn't even get to run, this is probably a failure related
+            # to imports in the test file
+            break
+
+        # Read what just failed
+        with open(stepcurrent_file) as f:
```

Comment:
Looks fine, but a more Pythonic(and atomic) way would be just to catch exception (as potentially something can delete the file between lines 598  and 604 )